src/dao/github/installation.js

```diff
@@ -0,0 +1,27 @@
+const BaseDao = require('~/src/dao').BaseDao
+const Installation = require('~/src/models/github/Installation')
+
+let dao
+
+exports.createDao = function () {
+  dao = new BaseDao({
+    modelType: Installation
+  })
+}
+
+/**
+* Upserts a new Installation record in the database, do nothing on conflict
+* @param id {Number} - github installation id
+* @param app_id {Number} - github app id
+*/
+exports.upsert = async function ({ id, app_id }) {
+  return dao.upsert({ id, app_id })
+}
+
+exports.findById = async function (id) {
```

Comment:
I realize that this is just copied over from hooks. What do you about if a method is not being overridden in any way, we just reference the original:

```js
exports.findById = dao.findById
exports.close = dao.destroy
```

I don't feel strongly about this. Just a thought. I may actually prefer it the way that it is because it explicitly shows that the methods are `async`.

Reply:
Ideally, we would just be extending the BaseDao here and everything would be a passthrough. I'm not sure these extra layers provide much value.